doc/htmldoc/installation/cmake_options.rst

```diff
@@ -87,6 +87,9 @@ External libraries
 +-------------------------------------------------------+------------------------------------------------------------------------------------------------+
 | ``-Dwith-gsl=[OFF|ON|</path/to/gsl>]``                | Build with the GSL library [default=ON]. To set a specific library, give the install path.     |
 +-------------------------------------------------------+------------------------------------------------------------------------------------------------+
++-------------------------------------------------------+------------------------------------------------------------------------------------------------+
+| ``-Dwith-hdf5=[OFF|ON|</path/to/hdf5>]``              | Build with HDF5 library [default=OFF]. To set a specific library, give the install path.       |
++-------------------------------------------------------+------------------------------------------------------------------------------------------------+
```

Comment:
Here (and also for GSL above and probably other options), we only explain technically how to build NEST with a given library. But we say nothing about the purpose or actual effect. Should we mention here that HDF5 is needed for Sonata, and similarly for GSL that it is needed for a range of neuron models? @jessica-mitchell @jougs Comments?

Reply:
I suggest we improve this in a separate PR. I have created issue https://github.com/nest/nest-simulator/issues/2765 to remind us of the needed improvements.

Reply:
Yes I think this makes sense to add more context. We could, in my opinion, add some relevant text here for SONATA, and in a separate PR look at adding more context to all the options.